elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/relayjsonapi/RelayJsonApiDSL.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.testhelpers.relayjsonapi;
+
+import com.yahoo.elide.contrib.testhelpers.jsonapi.elements.Attribute;
+import com.yahoo.elide.contrib.testhelpers.jsonapi.elements.Attributes;
+import com.yahoo.elide.contrib.testhelpers.relayjsonapi.elements.Data;
+import com.yahoo.elide.contrib.testhelpers.relayjsonapi.elements.Edges;
+import com.yahoo.elide.contrib.testhelpers.relayjsonapi.elements.Node;
+import com.yahoo.elide.contrib.testhelpers.relayjsonapi.elements.Resource;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+public class RelayJsonApiDSL {
```

Comment:
Why is JsonApi in the name of the DSL?

Reply:
This class has been deleted but I gave it JsonApi because its data has the JSON format.